### ISSUE TITLE
IMAP pull 24h -> pulls more often (at least every 2 hours)

### ISSUE THREAD

**Issue by Maetts** [k9mail/k-9#4704]
Please search to check for an existing issue (including closed issues, for which the fix may not have yet been released) before opening a new issue: https://github.com/k9mail/k-9/issues?q=is%3Aissue

### Expected behavior
If set to pull only every 24h it should pull mails only every 24h.

### Actual behavior
It pulls mails more often than only once a day.

### Steps to reproduce
1. Set up mail account
2. Set up imap pull to 24h
3. Wait for pull

### Environment
K-9 Mail version: 5.708/710

Android version: 10

Account type (IMAP, POP3, WebDAV/Exchange): IMAP

Please take some time to [retrieve logs](https://github.com/k9mail/k-9/wiki/LoggingErrors) and attach them here: No logs available, Network settings auto sync and always

**Comment by Maetts** [comment]
@cketti is there anything new or is there no further action on this?
